contracts/interfaces/ICircuitValidator.sol

```diff
@@ -2,21 +2,12 @@
 pragma solidity 0.8.16;
 
 interface ICircuitValidator {
-    struct CircuitQuery {
-        uint256 schema;
-        uint256 claimPathKey;
-        uint256 operator;
-        uint256[] value;
-        uint256 queryHash;
-        string circuitId;
-    }
-
     function verify(
         uint256[] memory inputs,
         uint256[2] memory a,
         uint256[2][2] memory b,
         uint256[2] memory c,
-        uint256 queryHash
+        bytes calldata circuitQueryData
     ) external view returns (bool r);
 
     function getCircuitId() external pure returns (string memory id);
```

Comment:
remove function getChallengeInputIndex() method, 
add method 
https://github.com/AndriianChestnykh/EIPs/blob/standard-interface-for-ZK-verification/EIPS/eip-standard-interface-for-zk-verification.md#:~:text=function%20inputIndexOf(string%20memory%20name)%20external%20view%20returns%20(uint256)%3B and implement in validators